backends-clickhouse/src/test/scala/io/glutenproject/execution/GlutenClickHouseTPCHParquetSuite.scala

```diff
@@ -2227,5 +2227,28 @@ class GlutenClickHouseTPCHParquetSuite extends GlutenClickHouseTPCHAbstractSuite
     compareResultsAgainstVanillaSpark(select_sql, true, { _ => })
     spark.sql("drop table test_tbl_3521")
   }
+
+  test("GLUTEN-3135 revert: Bug fix to_date") {
```

Comment:
please use ut `GLUTEN-3135: Bug fix to_date`, re-open it, don't create one again.

Reply:
done